packages/desktop/views/dashboard/wallet/panes/account-summary/components/AccountNetworkSummary.svelte

```diff
@@ -93,12 +95,21 @@
         </account-network-summary-balance-secondary>
     </account-network-summary-balance>
     <account-network-summary-assets class="flex flex-row justify-between items-center">
-        <Avatar backgroundColor="indigo-950" shape="square" text={nftCountFormatted} />
-        <Avatar backgroundColor="indigo-950" text={tokenCountFormatted} />
+        <Avatar
+            backgroundColor={dark ? 'surface-invert-dark' : 'surface-invert'}
+            shape="square"
+            text={nftCountFormatted}
+        />
+        <Avatar backgroundColor={dark ? 'surface-invert-dark' : 'surface-invert'} text={tokenCountFormatted} />
     </account-network-summary-assets>
 </account-network-summary>
 
 <style lang="postcss">
+    account-network-summary {
+        @apply bg-surface dark:bg-surface-dark;
+        @apply divide-y divide-solid divide-stroke dark:divide-stroke-dark;
```

Comment:
```suggestion
        @apply h-full w-full;
        @apply flex flex-col justify-between;
        @apply bg-surface dark:bg-surface-dark;
        @apply divide-y divide-solid divide-stroke dark:divide-stroke-dark;
```

+ remove the class on the html tag